Day2/BoxParser.py

```diff
@@ -0,0 +1,29 @@
+class BoxParser:
+    
+    raw_dimensions = None
+    parsed_dimensions = []
+    
+    def __init__(self, raw_dimensions):
+        self.raw_dimensions = raw_dimensions
+        self.parsed_dimensions = []
+    
+    def parse_single_box(self, box, delimiter="x"):
+        split_dimensions = box.split(delimiter)
+        int_dimensions = tuple(int(s) for s in split_dimensions)
+        return int_dimensions
+    
+    def is_tenth_line(self, floor, rob_wants_skipping=False):
+        if rob_wants_skipping:
+            return floor == 10
+        else:
+            return False
+    
+    def parse_all_boxes(self):
+        for i, box in enumerate(self.raw_dimensions, 1):
+            if self.is_tenth_line(i):
```

Comment:
ezt megfordinanam:
ha nem 10th line akkor parse meg append

de ez alapvetoen bugos :D sose fogod elerni a continuet mert is_tenth_line(i, false) amivel meghivodik es ha false a rob_wants_skipping akkor false a return mindig

ahogy a robi mondta default valuek nem a legjobbak, de most meg korai hasznalnod oket mert elfedkezel roluk 2bol 2szer lett bug, ha nem default vaule akkor sir a IDE es vegig kell gondolnod h melyiket is akarod